integration_tests/testsuite/deploy_app.py

```diff
@@ -76,17 +80,67 @@ def _record_latency_to_bigquery(deploy_latency, language, is_xrt):
 def deploy_app_and_record_latency(appdir, language, is_xrt):
     start_time = time.time()
 
-    version = deploy_app(None, None, appdir, None)
+    version, url = deploy_app(None, None, appdir, None, 'GAE')
```

Comment:
Could you use the constant instead of 'GAE'?

Reply:
done